.github/workflows/run-e2e-suite.yaml

```diff
@@ -30,6 +30,7 @@ env:
   AZURE_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
   AZURE_CLIENT_SECRET: ${{ secrets.AZURE_CLIENT_SECRET }}
   AZURE_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
+  KUBERNETES_VERSION: "v1.26.3"
```

Comment:
I'd favour an explicit value in the e2e config that specifies the k8s version of the management cluster.

Reply:
Fixed, now we have `KUBERNETES_MANAGED_VERSION` for that.